CHANGELOG.md

```diff
@@ -8,6 +8,53 @@ Changes since last non-beta release.
 
 *Please add entries here for your pull requests.*
 
+## 9.0 from 8.x. Upgrade Instructions
+All 9.0.0 beta versions can be viewed in [PR 908](https://github.com/shakacode/react_on_rails/pull/908)
+
+- Update the gemfile. Switch over to using Webpacker on the ShakaCode branch:
+
+```rb
+gem "webpacker", git: "https://github.com/shakacode/webpacker.git",
+    branch: "issue-464-merge-webpacker-lite-into-webpacker"
+```
+
```

Comment:
That's done for every version update. Do you feel this is worth mentioning here?